README.md

```diff
@@ -15,3 +15,50 @@ The API service is a gRPC server that offers a interface to fetched the stored h
 ### Gateway
 
 The gateway service is main entry point for third parties to access all other systems. Currently, it is responsible for proxying requests to the API service
+
+## Local Kubernetes setup
+
+- Install depedencies:
+    - [Docker](https://docs.docker.com/get-docker/) 
+    - [Minikube](https://minikube.sigs.k8s.io/docs/start/)
+    - kubectl `brew install kubectl`
+    - kubesec `brew install shyiko/kubesec/kubesec`
+    - gpg `brew install gnupg`
+- Create a `GPG` key
+    - `gpg --full-generate-key`
+    - Answers the questions
+    - List your `GPG` keys
+        - `gpg --list-secret-keys --keyid-format=long`
+- Start Minikube 
+    - `minikube start`
+- Change the docker daemon your terminal points at 
+    - `eval $(minikube -p minikube docker-env)`
+    - This only changes your current terminal session. By closing the terminal, you will go back to using your own system’s docker daemon.
```

Comment:
Do you need minikube? We normally just use docker desktop

Reply:
This mainly came down to me not using the Docker desktop k8s and knowing how to teardown minikube so I could test my commands worked. I could look into Docker desktop as a separate task? This PR was mainly to play around with k8s.